.github/release/full/Dockerfile

```
@@ -45,7 +49,7 @@ COPY --from=build-step /usr/local/lib /usr/local/lib
 COPY dist/wetterdienst-*.whl /tmp/
 
 # Install latest wheel package.
-RUN pip install $(ls -c /tmp/wetterdienst-*-py3-none-any.whl)[http,sql,export,duckdb,influxdb,cratedb,mysql,postgresql,radar,bufr,ui]
+RUN pip install $(ls -c /tmp/wetterdienst-*-py3-none-any.whl)[http,sql,export,duckdb,influxdb,cratedb,mysql,postgresql,radar,bufr,ui,levenstein]
```

There's a typo here and at the two following occurrances.

```suggestion
RUN pip install $(ls -c /tmp/wetterdienst-*-py3-none-any.whl)[http,sql,export,duckdb,influxdb,cratedb,mysql,postgresql,radar,bufr,ui,levenshtein]
```